src/DistributedLock.Postgres/PostgresAdvisoryLockKey.cs

```diff
@@ -94,12 +94,12 @@ internal long Key
     public bool Equals(PostgresAdvisoryLockKey that) => this.ToTuple().Equals(that.ToTuple());
 
     /// <summary>
-    /// Implements <see cref="Object.Equals(object)"/>
+    /// Implements <see cref="object.Equals(object)"/>
     /// </summary>
-    public override bool Equals(object obj) => obj is PostgresAdvisoryLockKey that && this.Equals(that);
+    public override bool Equals(object? obj) => obj is PostgresAdvisoryLockKey that && this.Equals(that);
```

Comment:
@davidngjy this was the fix for the nullability error. The initial nullability releases had some of the wrong nullability typings on core methods, so the introduction of .net8 forced us to update to the correct typings.